pkg/services/ngalert/api/util.go

```diff
@@ -116,7 +116,10 @@ func (p *AlertingProxy) withReq(
 			var m map[string]interface{}
 			if err := json.Unmarshal(resp.Body(), &m); err == nil {
 				if message, ok := m["message"]; ok {
-					errMessage = message.(string)
+					errMessageStr, isString := message.(string)
```

Comment:
I spent about an hour trying to create a test but the dependencies on `	DataProxy *datasourceproxy.DataSourceProxyService` makes this way too complex for what we're actually fixing. I could have split this function is half to have a more isolated test but I see little to no value in this given the problem is pretty obvious.

If you strongly think we should have one, I'm happy to give it another go.

Reply:
Sounds sensible to me, the problem is obvious enough 👍